.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[Bug] : "
+labels: bug, triage
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+<!-- **Please add a reproducible repo or sandbox** -->
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem. (from [cssnano-playground](https://cssnano.co/playground/))
+
+**Desktop (please complete the following information):**
+- CSSNANO Version [e.g. 22] 
+- Plugin version (if it's known)
+- Add the output of `npx envinfo`(recommended) or `npm ls cssnano` here
```

Comment:
We need to have an one command and developers should put all information here, it is very bad to write `if it's known`

Reply:
yes, does makes sense.
removing that .